doc/api/os.md

```diff
@@ -202,6 +202,8 @@ added: v2.3.0
 
 The `os.homedir()` method returns the home directory of the current user as a
 string.
+Calling this might rarely result in an exception due to e.g. `ENOENT` and other
```

Comment:
`ENOBUFS` and `EINVAL` **shouldn't** be returned to Node, given that we control how `uv_os_homedir()` is called. I would say that telling JavaScript developers to worry about that is more trouble than it's worth. `ENOMEM` means that the application is out of memory. It is probably not worth trying to catch that error either, as you're likely to crash soon.

> Do you think it makes sense to document something like that in a more generic way?

Not really.

> I expect quite a few people not to think about that when they can't find anything like that in the documentation.

I don't think there have been many bug reports for things like this related to libuv calls. If a bug report is opened for something like this, it's probably a legitimate bug.